### PR TITLE
Bulk upload

### PR DESCRIPTION
**Pull request by Khukhunashvili**
**Part of GCI 2018** :tada: :balloon: 

preview

### CODE REVIEW

File: src/app/organization/organization.component.html [hunk 1]
```diff
@@ -14,12 +14,21 @@ <h4> Download </h4>
     <div class="container-fluid">
       <div class="row action-wrapper">
         <div class="col-2"></div>
-        <button class="col-lg-3"> IMPORT </button>
+        <input type="file" class="d-none" accept=".csv" #bulkUploader (change)="onFileUpload($event)">
+        <button class="col-lg-3" (click)="bulkUploader.click()"> IMPORT </button>
```
Comment (nikitadhiman) [inline src/app/organization/organization.component.html]: If it's (click) why do you need to call bulluploader.click again?

Comment (Khukhuna) [reply src/app/organization/organization.component.html]: ```<Input type="file">``` looks like this:

image


It was breaking the UI, so instead I attached **_import_** button to invisible inputs **_browse_** button.
Now it looks like 'Import' button is uploading files.

Comment (nikitadhiman) [reply src/app/organization/organization.component.html]: I would recommend you can look for better ways to do this, on (click) again a .click is not at all optimal

Comment (Khukhuna) [reply src/app/organization/organization.component.html]:
```html
        <label class="col-lg-3 main-button text-center">
            <input type="file" class="d-none"/>
            IMPORT
        </label>
```

Wrapped ```html <input type="file"> ``` in label tag.
Functionality & UI is same. :smile:

**Comment by nikitadhiman** [comment]
great!

**Comment by nikitadhiman** [comment]
Hi, can you please resolve the conflicts on this. thanks

**Comment by nikitadhiman** [comment]
Great work!! merging this

**Pull request merged**